setup.py

```diff
@@ -430,7 +430,7 @@ def build_extension(self, ext: Extension):
         destination = os.path.dirname(os.path.abspath(self.get_ext_fullpath(ext.name)))
         subprocess.check_call(["go", "install", "golang.org/x/tools/cmd/goimports"],
                               env={"PATH": bin_path, **go_env})
-        subprocess.check_call(["go", "get", "github.com/go-python/gopy@v0.4.0"],
+        subprocess.check_call(["go", "get", "github.com/go-python/gopy@v0.4.4"],
```

Comment:
Have we updated the build process to pass in the new flags? If not, this change is incomplete (and should be in a different PR anyway).

Reply:
yup